src/Control/Email/Mailer.php

```diff
@@ -2,526 +2,17 @@
-    {
-        if (!$file) {
-            throw new InvalidArgumentException("Not passed a filename and/or data");
-        }
-
-        if (is_string($file)) {
-            $file = array('filename' => $file);
-            $fh = fopen($file['filename'], "rb");
-            if ($fh) {
-                $file['contents'] = "";
-                while (!feof($fh)) {
-                    $file['contents'] .= fread($fh, 10000);
-                }
-                fclose($fh);
-            }
-        }
-
-        // Build headers, including content type
-        if (!$destFileName) {
-            $base = basename($file['filename']);
-        } else {
-            $base = $destFileName;
-        }
-
-        $mimeType = !empty($file['mimetype']) ? $file['mimetype'] : HTTP::get_mime_type($file['filename']);
-        if (!$mimeType) {
-            $mimeType = "application/unknown";
-        }
-        if (empty($disposition)) {
-            $disposition = isset($file['contentLocation']) ? 'inline' : 'attachment';
-        }
-
-        // Encode for emailing
-        if (substr($mimeType, 0, 4) != 'text') {
-            $encoding = "base64";
-            $file['contents'] = chunk_split(base64_encode($file['contents']));
-        } else {
-            // This mime type is needed, otherwise some clients will show it as an inline attachment
-            $mimeType = 'application/octet-stream';
-            $encoding = "quoted-printable";
-            $file['contents'] = quoted_printable_encode($file['contents']);
-        }
-
-        $headers =  "Content-type: $mimeType;\n\tname=\"$base\"\n".
-                    "Content-Transfer-Encoding: $encoding\n".
-                    "Content-Disposition: $disposition;\n\tfilename=\"$base\"\n";
-
-        if (isset($file['contentLocation'])) {
-            $headers .= 'Content-Location: ' . $file['contentLocation'] . "\n" ;
-        }
-
-        $headers .= $extraHeaders . "\n";
-
-        // Return completed packet
-        return $headers . $file['contents'];
-    }
-
-    /**
-     * Cleans up emails which may be in 'Name <email@silverstripe.com>' format
-     *
-     * @param string $emailAddress
-     * @return string
-     */
-    protected function validEmailAddress($emailAddress)
-    {
-        $emailAddress = trim($emailAddress);
-        $openBracket = strpos($emailAddress, '<');
-        $closeBracket = strpos($emailAddress, '>');
-
-        // Unwrap email contained by braces
-        if ($openBracket === 0 && $closeBracket !== false) {
-            return substr($emailAddress, 1, $closeBracket - 1);
-        }
-
-        // Ensure name component cannot be mistaken for an email address
-        if ($openBracket) {
-            $emailAddress = str_replace('@', '', substr($emailAddress, 0, $openBracket))
-                . substr($emailAddress, $openBracket);
-        }
-
-        return $emailAddress;
-    }
+    public function getFailedRecipients();
```

Comment:
Was this going to be moved to Email? `SwiftMailer` doesn't implement this, so we can just remove this I guess.